[Documentation Fix] Building mkl_interface example is failing as it can't find example_helper.hpp

Building the C++ example in https://github.com/IntelPython/example-portable-data-parallel-extensions/blob/main/steps/mkl_interface/building.md#building-a-c-example-that-uses-the-library fails with

```bash
> icpx -fsycl getrs_usm.cpp -fsycl-targets=nvptx64-nvidia-cuda,spir64-unknown-unknown -I${MKL_INTERFACE_ROOT}/include -L${MKL_INTERFACE_ROOT}/lib -lonemkl -Wl,-rpath,${MKL_INTERFACE_ROOT}/lib -o run -I -I${MKL_INTERFACE_ROOT}/../examples/include
getrs_usm.cpp:48:10: fatal error: 'example_helper.hpp' file not found
   48 | #include "example_helper.hpp"
      |          ^~~~~~~~~~~~~~~~~~~~
1 error generated.
```

I added `-I${MKL_INTERFACE_ROOT}/../examples/include` to the icpx command and that seems to have worked.

--- a/steps/mkl_interface/building.md
+++ b/steps/mkl_interface/building.md
@@ -68,6 +68,7 @@ cd examples/lapack/run_time_dispatching/
 icpx -fsycl getrs_usm.cpp                                     \
     -fsycl-targets=nvptx64-nvidia-cuda,spir64-unknown-unknown \
     -I${MKL_INTERFACE_ROOT}/include                           \
+    -I${MKL_INTERFACE_ROOT}/../examples/include               \
     -L${MKL_INTERFACE_ROOT}/lib -lonemkl                      \
     -Wl,-rpath,${MKL_INTERFACE_ROOT}/lib -o run
 ```

